auto.def

```diff
@@ -110,6 +110,8 @@ options {
   with-zstd:path            => "Location of Zstandard"
 # System
   with-sysroot:path         => "Target system root"
+# Misc
+  include-path-in-cflags=1  => "Remove include paths from CFLAGS in the output of neomutt -v"
```

Comment:
```suggestion
  include-path-in-cflags=1  => "Show include paths in CFLAGS in the output of neomutt -v"
```

To match the wording of the configure option.

Reply:
autoconf will show the contrary of the default in the help screen:

```
./configure --help | grep cflags
  --disable-include-path-in-cflags  Remove include paths from CFLAGS in the output of neomutt -v
```

Reply:
ah!  that's smart